### ISSUE TITLE
Or condition in where does not work as expected if one child is `level == None`

### ISSUE THREAD

**Issue by fabiencelier** [atoti/atoti#428]
I want a measure which is only expressed when one level is None or on a specific value.
I use a OR condition in my where but it does not behave as expected.

## Steps to reproduce
```python
import pandas as pd, atoti as tt
session = tt.create_session()
base = session.read_pandas(
    pd.DataFrame({ "ID": [1,2,3], "Product": ["a", "b","c"], "Amount": [10.0, 20.0, 30.0]}),
    table_name = "sales"
)
cube = session.create_cube(base)
m,l,h = cube.measures, cube.levels, cube.hierarchies
```
Consider the following 3 measures

```python
m["where null"] = tt.where(l["Product"] == None, 1, 0)
m["where a"] = tt.where(l["Product"] == "a", 1, 0)
m["where null or a"] = tt.where((l["Product"] == None) | (l["Product"] == "a"), 1, 0)
```

## Actual Result

```python
cube.query(m["where null"], m["where a"], m["where null or a"], levels=[l["Product"]], include_totals=True)
```

![image](https://user-images.githubusercontent.com/17087544/144246820-53dd2f1f-84ae-4e84-9da5-22694d67817d.png)

## Expected Result

I'm not sure if that's a bug or by design but I expected `m["where null or a"]` to be 1 on the grand total and not `None`.

 A workaround is to define a measure with several where:

```python
m["multi where"] = tt.where(
    l["Product"] == None,
    1,
    tt.where(l["Product"] == "a", 1, 0))
```

## Environment

- atoti: 0.6.4
- Python: 3.8.5
- Operating System: linux

**Comment by tibdex** [comment]
Thanks for reporting this and for suggesting a workaround 👍.
Like you, I would expect `m["where null or a"]` and `m["multi where"]` to be equivalent.

Looking into it briefly, the difference comes from the fact that `l["Product"] == None)` is a `LevelCondition` while `(l["Product"] == None) | (l["Product"] == "a")` is a `BooleanMeasure` with the operator `or` and its first operand as another `BooleanMeasure` this time with the operator `isNull`.

**Comment by Skyscimitar** [comment]
Linking internal issue where we decided to remove required levels only for null comparisons: https://github.com/activeviam/atoti/issues/1918

**Comment by bloussou** [comment]
Hello,
This is fixed in atoti `0.7.0` so I'm closing this issue.
![image](https://user-images.githubusercontent.com/32448554/186155635-02f85106-25f4-4ecc-a9aa-f4abd8ec5a18.png)
